test_vector_handlers/compatibility-requirements/1.3.3

```diff
@@ -0,0 +1,2 @@
+aws-encryption-sdk==1.3.3
+attrs<19.2.0
```

Comment:
Why did we add a version for `attrs` here?

Reply:
https://github.com/aws/aws-encryption-sdk-python/pull/196/commits/03baa4903c772ee42514f400de92e66caea7a9f5

Reply:
bah...didn't expand the comment :(

> attrs==19.2.0 removed a deprecated feature that aws-encryption-sdk==1.3.3 depended on.
This reorganization lets us define specific requirements bounds for old versions of aws-encryption-sdk
that will probably continue to be necessary as these old versions age.